pages/public_cloud/containers_orchestration/managed_kubernetes/expose_your_applications_using_a_load_balancer/guide.en-gb.md

```diff
@@ -411,35 +411,9 @@ test-lb-todel        LoadBalancer   10.3.107.18   141.94.215.240   80:30172/TCP
 
 When exposing services like nginx-ingress-controller, it's a common requirement that the client connection information could pass through proxy servers and load balancers, therefore visible to the backend services. Knowing the originating IP address of a client may be useful for setting a particular language for a website, keeping a denylist of IP addresses, or simply for logging and statistics purposes. You can follow the official Cloud Controller Manager documentation on how to [Use PROXY protocol to preserve client IP](https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/openstack-cloud-controller-manager/expose-applications-using-loadbalancer-type-service.md#use-proxy-protocol-to-preserve-client-ip).
 
-#### Migrate from Loadbalancer for Kubernetes to Public Cloud Load Balancer
-
-In order to migrate from an existing [Loadbalancer for Kubernetes](/links/public-cloud/load-balancer-kubernetes) to a [Public Cloud Load Balancer](/links/public-cloud/load-balancer) you will have to modify an existing Service and change its LoadBalancer class.
-
-Your existing LoadBalancer Service using [Loadbalancer for Kubernetes](/links/public-cloud/load-balancer-kubernetes) should have the following annotation:
-
-```yaml
-annotations:
-  loadbalancer.ovhcloud.com/class: "iolb"
-```
-
-##### Step 1 - Edit your Service to change the LoadBalancer class to 'octavia'
-
-```yaml
-annotations:
-  loadbalancer.ovhcloud.com/class: "octavia" // not required for clusters running kubernetes versions >= 1.31, you can just remove the annotation.
-```
-
-##### Step 2 - Apply the change
-
-```yaml
-kubectl apply -f your-service-manifest.yaml
-```
-
 > [!warning]
 >
-> As [Loadbalancer for Kubernetes](/links/public-cloud/load-balancer-kubernetes) and [Public Cloud Load Balancer](/links/public-cloud/load-balancer) do not use the same solution for Public IP allocation, **it is not possible to keep the existing public IP** of your Loadbalancer for Kubernetes.
-> Changing the LoadBalancer class of your Service will lead to the creation of a new Loadbalancer and the allocation of a new Public IP (Floating IP).
->
+> Only ProxyProtocol version 1 is supported at the moment by the MKS's integration.
```

Comment:
This is wrong, I mitigate this restriction few week ago. (we now support v1 & v2)